### ISSUE TITLE
A DOM node with an ID value "Group_4719" already exists!

### ISSUE THREAD

**Issue by CreateSean** [putyourlightson/craft-sprig#40]
### Describe the bug

I had a load more component working previously. Today I went to that page to do some mobile QA (site isn't live yet) and it's broken with this error.
```
A DOM node with an ID value "Group_4719" already exists!
1. in /var/www/html/vendor/ivopetkov/html5-dom-document-php/src/HTML5DOMDocument.php
```

The error disappears if I comment out the sprig component
```
    {{ sprig('_components/load-more', {limit: 3, offset: 6, featuredId: featured.id}, {class: 'container flex flex-row flex-wrap w-full'}) }}
```

I have updated craft and plugins since the last time I looked at this page so it may be related to the latest version of Sprig or Craft.

Please advise

### To reproduce
Steps to reproduce the behaviour:
1. Go to "..."
2. Click on "..."
3. See error message "..."

### Expected behaviour

I expect the component to load when clicking load more like it was previously

### Versions
- Plugin version: 1.0.0-beta.17
- Craft version:3.5.12.1

**Comment by bencroker** [comment]
This has already been fixed for the next release, but you should regardless try to avoid having elements with the same ID in your markup.
https://github.com/putyourlightson/craft-sprig/blob/develop/CHANGELOG.md#100-beta18---unreleased

**Comment by bencroker** [comment]
Released in 1.0.0-beta.18.
